ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -362,6 +362,18 @@ class RangeSelector {
             this.dropdown.selectedIndex = i + 1;
         }
 
+        const resetFixedRange = (): void => {
+            // If rangeSelector.selected is larger than actual range, reset
+            // chart fixed range (#20327)
+            const xAxis = chart.xAxis[0];
+            if (xAxis.dataMax && xAxis.dataMin && rangeOptions._range) {
```

Comment:
When checking numbers, be careful with truthy values (`dataMax` or `dataMin` can be 0)

Reply:
Rookie mistake, thanks 😬